code/src/index.css

```diff
@@ -1,13 +1,826 @@
+  background: tomato;  /* background color in case image doesnt load */
+  width: 407px;
+  height: 575px;
+  opacity: 0.9;
+  border: solid black 1px;
+}
+
+.norma {
+  background: tomato;  /* background color in case image doesnt load */
+  width: 380px;
+  height: 885px;
+  opacity: 0.95;
+  border: solid black 1px;
+}
+
+.amatriciana {
+  background: tomato;  /* background color in case image doesnt load */
+  width: 407px;
+  height: 717px;
+  opacity: 0.95;
+  border: solid black 1px;
+}
+
+.img-with-text {
+  display: flex;
+  flex-direction: column;
+  align-self: flex-end;
+  text-align: justify;
+  width: 374px;
+  padding-left: 50px;
+}
+
+.img-with-text img {
+  display: block;
+  margin: 0 auto;
+}
+
+.ingredients {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+}
+
+.instructions {
+  text-align: left;
+}
+
+.note {
+  margin-bottom: 0px;
+}
+
+.extra {
+  width: 800px;
+  text-align: left;
+  padding-left: 30px;
+}
+
+/* RECIPE FOOTER */
+
+.footer {
+  width: 100%;
+}
+
+.footer-container {
+  display: flex;
+  flex-direction: column;
+  justify-self: center;
+  margin: 70px 0px 10px 0px;
+}
+
+.footer-heading {
+  font-size: 11px;
+  font-weight: bold;
+  margin: 40px 0px 0px 0px;
+}
+
+.footer-text {
+  font-size: 11px;
+  font-weight: bold;
+}
+
+/* RECIPE TEXT CLASSES */
+
+.sub-heading {
+  margin-bottom: 25px;
+  font-size: 20px;
+  font-weight: bolder;
+}
+
+.extra-heading {
+  margin-bottom: 15px;
+  font-size: 21px;
+  font-weight: bolder;
+}
+
+.image-header {
+  font-size: 12px;
+  font-weight: bold;
+  margin-bottom: 0px;
+}
+
+.image-text {
+  font-size: 10px;
+  font-weight: normal;
+  margin: 0px;
+}
+
+.ingredient {
+  margin: 1px;
+}
+
+.p-norma {
+  margin-bottom: 0px;
+}
+
+/* MOBILE */
+@media (max-width: 667px) {
+
+.landing-page {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.landing-container {
+  /* border: solid #66ca6b 1px; */
+  display: flex;
+  flex-direction: column;
+}
+
+.landing-heading {
+  font-size: 50px;
+  text-align: center;
+}
+
+.landing-button {
+  padding: 15px;
+  font-size: 13px;
+  width: 135px;
+  align-self: center;
+}
+
+.landing-button:hover {
+  background-color: #66ca6b;
+  border: solid white 1px;
+}
+
+.container {
+  padding: 20px;
+  width: 75vw;
+  height: fit-content;
+  align-self: center;
+}
+
+/* QUESTIONS MOBILE */
+
+input[type="text" i] {
+  text-align: center;
+  padding: 5px 2px;
+  border: none;
+  border-radius: 2px;
+  font-family: 'Noto Sans Mono', monospace;
+  font-size: 10px;
+  font-weight: normal;
+}
+
+.button {
+  padding: 10px 25px;
+  margin-bottom: 40px;
+  font-size: 13px;
+}
+
+.radio-label {
+  font-size: 12px;
+  font-weight: bold;
+}
+
+/* TEXT STYLING MOBILE */
+
+h1 {
+  font-size: 19px;
+  margin: 30px 0px 30px 0px;
+}
+
+h2 {
+  font-size: 17px;
+}
+
+h3 {
+  font-size: 15px;
+}
+
+p {
+  font-size: 9px;
+}
+
+li {
+  font-size: 9px;
+}
+
+/* RECIPE MOBILE */
+
+.recipe {
```

Comment:
There is some repetition here from .recipe on line 201, there's some things that are identical and can be removed :)